helm/docs-app/values.yaml

```diff
@@ -1,10 +1,5 @@
 name: docs-app
 namespace: docs
-image:
```

Comment:
With PSP removed, you can also remove this block from below:

```yaml
global:
  podSecurityStandards:
    enforced: false
```

Then regenerate the values schema.

Reply:
done